app/bundles/EmailBundle/Controller/EmailController.php

```diff
@@ -1475,6 +1477,92 @@ public function batchDeleteAction(Request $request): Response
         );
     }
 
+    public function scheduleSendAction(CorePermissions $security, EmailModel $model, Request $request, int $objectId): JsonResponse|Response
+    {
+        /** @var Email $entity */
+        $entity = $model->getEntity($objectId);
```

Comment:
The typehint is wrong. It can be null. And the `EmailModel::getEntity()` method has correct return type. Can we remove this?
```suggestion
        $entity = $model->getEntity($objectId);
```